core/src/sumeragi/network_topology.rs

```diff
@@ -320,47 +268,28 @@ pub enum Role {
     Undefined,
 }
 
-/// Error during signature verification
-#[derive(thiserror::Error, displaydoc::Display, Debug, Clone, Copy, PartialEq, Eq)]
-pub enum SignatureVerificationError {
-    /// The block doesn't have enough valid signatures to be committed ({votes_count} out of {min_votes_for_commit})
-    NotEnoughSignatures {
-        /// Current number of signatures
-        votes_count: usize,
-        /// Minimal required number of signatures
-        min_votes_for_commit: usize,
-    },
-    /// The block doesn't contain an expected signature. Expected signature can be leader or the current peer
-    SignatureMissing,
-    /// Found signature that does not correspond to block payload
-    UnknownSignature,
-    /// The block doesn't have proxy tail signature
-    ProxyTailMissing,
-    /// The block doesn't have leader signature
-    LeaderMissing,
-}
-
 #[cfg(test)]
 mod tests {
     use iroha_crypto::KeyPair;
+    use iroha_primitives::unique_vec;
 
     use super::*;
-    use crate::block::ValidBlock;
 
-    macro_rules! peers {
+    #[macro_export]
```

Comment:
exported for what reason?

Reply:
there are other ways to make it accessible from `block.rs`

Reply:
Oof, yes. Working at late nights have its own consequences